src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
+
+.bigSearch {
+  margin: 20px 0;
+}
+
+.searchInput {
+  width: 100%;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  outline: none;
+}
+
+.bigSearchInput {
+  height: 70px;
+  padding: 0 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+  font-size: 16px;
+}
+
+.smallSearchInput {
+  height: 42px;
+  padding: 0 33px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 12px 14px;
+  font-size: 14px;
+}
+
+.searchInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigSearchInput:focus {
+  font-weight: bold;
+  text-shadow: none;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: url(images/Search.svg) no-repeat 26px 25px / 19px, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.smallSearchInput:focus {
```

Comment:
You can move similar properties for these classes to searchInput:focus, and leave only background property for bigSeatchInput and smallSearchInput

Reply:
yeah, thanks! did it